README.md

```diff
@@ -1,10 +1,18 @@
-Although we are working heavily to make this demo public, there are many tasks yet to be accomplished.
+## Notice
+This project is still in under construction. We plan to release it in early March.
 
-Please come back after we clean up repository and verify everything is consistent.
-It is plan to be ready in early March.
-
----
 ![](./setup/image/demo_overview.png)
+"Find Your Candy with Cloud ML" system diagram
+
+## Understanding your request with ML APIs
+The demo starts with listening to your voice request such as "I like a dark chewy chocolate", and analyzes it with Cloud Speech API. The API converts audio data into text with Google's high quality voice recognition technology. Then the text is processed with Cloud Natural Language API and its syntactic analysis so that the system can extract what are the important verbs, adjectives and nouns in the text to understand the meaning of your command.
+
+Then the system uses word2vec algorithm running on Cloud Machine Learning (Cloud ML) for choosing the best candy for your request. The algorithm is smart enough to understand the similarity of meaning between two words - such as "milky and creamy", "hot and spicy" - based on an analysis result on natural languages. With this technology, the system tries to recommend a candy that has the highest likelihood of fulfilling the request.
```

Comment:
We want to write 
- "Then the system uses word2vec and regression(inception-v3 + transfer learning) on Cloud Machine Learning(CloudML)"
instead of 
- "Then the system uses word2vec algorithm running on Cloud Machine Learning (Cloud ML) " .